Compilation error of CGAL::polyhedron_3_to_nef_3 with CGAL_NO_DEPRECATED_CODE

https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-4.10-Ic-35/Nef_3_Examples/TestReport_lrineau_Ubuntu-Latest-NO_DEPRECATED_CODE.gz
```
/usr/bin/c++   -DCGAL_TEST_SUITE -DCGAL_USE_GMP -DCGAL_USE_MPFR -I/home/cgal_tester/build/src/cmake/platforms/Ubuntu-Latest-NO_DEPRECATED_CODE/test/Nef_3_Examples/../../include -isystem /usr/include/x86_64-linux-gnu -I/home/cgal_tester/build/src/cmake/platforms/Ubuntu-Latest-NO_DEPRECATED_CODE/test/Nef_3_Examples -I/home/cgal_tester/build/src/cmake/platforms/Ubuntu-Latest-NO_DEPRECATED_CODE/include -I/mnt/testsuite/include  -DCGAL_NO_DEPRECATED_CODE -Wall -frounding-math   -o CMakeFiles/handling_double_coordinates.dir/handling_double_coordinates.cpp.o -c /home/cgal_tester/build/src/cmake/platforms/Ubuntu-Latest-NO_DEPRECATED_CODE/test/Nef_3_Examples/handling_double_coordinates.cpp
In file included from /usr/include/boost/utility/result_of.hpp:23:0,
                 from /mnt/testsuite/include/CGAL/result_of.h:29,
                 from /mnt/testsuite/include/CGAL/basic.h:40,
                 from /mnt/testsuite/include/CGAL/Cartesian/Cartesian_base.h:28,
                 from /mnt/testsuite/include/CGAL/Simple_cartesian.h:28,
                 from /mnt/testsuite/include/CGAL/Exact_predicates_exact_constructions_kernel.h:28,
                 from /home/cgal_tester/build/src/cmake/platforms/Ubuntu-Latest-NO_DEPRECATED_CODE/test/Nef_3_Examples/handling_double_coordinates.cpp:1:
/usr/include/boost/mpl/eval_if.hpp: In instantiation of 'struct boost::mpl::eval_if<boost::detail::has_vertex_property_type<CGAL::Polyhedron_3<CGAL::Epeck>, mpl_::bool_<false> >, boost::detail::get_vertex_property_type<CGAL::Polyhedron_3<CGAL::Epeck> >, boost::no_property>':
/usr/include/boost/graph/graph_traits.hpp:276:12:   required from 'struct boost::vertex_property_type<CGAL::Polyhedron_3<CGAL::Epeck> >'
/usr/include/boost/graph/properties.hpp:201:12:   required from 'struct boost::detail::vertex_property_map<CGAL::Polyhedron_3<CGAL::Epeck>, boost::face_external_index_t>'
/usr/include/boost/graph/properties.hpp:212:10:   required from 'struct boost::property_map<CGAL::Polyhedron_3<CGAL::Epeck>, boost::face_external_index_t, void>'
/mnt/testsuite/include/CGAL/Nef_3/polygon_mesh_to_nef_3.h:323:81:   required from 'void CGAL::polyhedron_3_to_nef_3(Polyhedron&, SNC_structure&) [with Polyhedron = CGAL::Polyhedron_3<CGAL::Epeck>; SNC_structure = CGAL::SNC_structure<CGAL::Epeck, CGAL::SNC_indexed_items, bool>]'
/mnt/testsuite/include/CGAL/Nef_polyhedron_3.h:605:55:   required from 'CGAL::Nef_polyhedron_3<K, I, Mk>::Nef_polyhedron_3(CGAL::Polyhedron_3<T1, T2, T3, T4>&) [with T1 = CGAL::Epeck; T2 = CGAL::Polyhedron_items_3; T3 = CGAL::HalfedgeDS_default; T4 = std::allocator<int>; Kernel_ = CGAL::Epeck; Items_ = CGAL::SNC_indexed_items; Mark_ = bool]'
/home/cgal_tester/build/src/cmake/platforms/Ubuntu-Latest-NO_DEPRECATED_CODE/test/Nef_3_Examples/handling_double_coordinates.cpp:15:21:   required from here
/usr/include/boost/mpl/eval_if.hpp:38:31: error: no type named 'type' in 'boost::mpl::eval_if<boost::detail::has_vertex_property_type<CGAL::Polyhedron_3<CGAL::Epeck>, mpl_::bool_<false> >, boost::detail::get_vertex_property_type<CGAL::Polyhedron_3<CGAL::Epeck> >, boost::no_property>::f_ {aka struct boost::no_property}'
     typedef typename f_::type type;
                               ^
In file included from /mnt/testsuite/include/CGAL/Nef_polyhedron_3.h:52:0,
                 from /home/cgal_tester/build/src/cmake/platforms/Ubuntu-Latest-NO_DEPRECATED_CODE/test/Nef_3_Examples/handling_double_coordinates.cpp:4:
/mnt/testsuite/include/CGAL/Nef_3/polygon_mesh_to_nef_3.h: In instantiation of 'void CGAL::polyhedron_3_to_nef_3(Polyhedron&, SNC_structure&) [with Polyhedron = CGAL::Polyhedron_3<CGAL::Epeck>; SNC_structure = CGAL::SNC_structure<CGAL::Epeck, CGAL::SNC_indexed_items, bool>]':
/mnt/testsuite/include/CGAL/Nef_polyhedron_3.h:605:55:   required from 'CGAL::Nef_polyhedron_3<K, I, Mk>::Nef_polyhedron_3(CGAL::Polyhedron_3<T1, T2, T3, T4>&) [with T1 = CGAL::Epeck; T2 = CGAL::Polyhedron_items_3; T3 = CGAL::HalfedgeDS_default; T4 = std::allocator<int>; Kernel_ = CGAL::Epeck; Items_ = CGAL::SNC_indexed_items; Mark_ = bool]'
/home/cgal_tester/build/src/cmake/platforms/Ubuntu-Latest-NO_DEPRECATED_CODE/test/Nef_3_Examples/handling_double_coordinates.cpp:15:21:   required from here
/mnt/testsuite/include/CGAL/Nef_3/polygon_mesh_to_nef_3.h:323:81: error: no type named 'type' in 'struct boost::property_map<CGAL::Polyhedron_3<CGAL::Epeck>, boost::face_external_index_t, void>'
   typedef typename boost::property_map<Polyhedron, face_external_index_t>::type FIMap;
                                                                                 ^
/mnt/testsuite/include/CGAL/Nef_3/polygon_mesh_to_nef_3.h:325:85: error: no type named 'type' in 'struct boost::property_map<CGAL::Polyhedron_3<CGAL::Epeck>, boost::halfedge_external_index_t, void>'
   typedef typename boost::property_map<Polyhedron, halfedge_external_index_t>::type HIMap;
                                                                                     ^
CMakeFiles/handling_double_coordinates.dir/build.make:62: recipe for target 'CMakeFiles/handling_double_coordinates.dir/handling_double_coordinates.cpp.o' failed
```

This is related to the merge of #1466, and only show up with `-DCGAL_NO_DEPRECATED_CODE`.

--- a/Nef_3/include/CGAL/Nef_polyhedron_3.h
+++ b/Nef_3/include/CGAL/Nef_polyhedron_3.h
@@ -53,6 +53,8 @@
 #include <CGAL/Nef_3/shell_to_nef_3.h>
 #include <CGAL/Polyhedron_incremental_builder_3.h>
 #include <CGAL/Polyhedron_3.h>
+#include <CGAL/boost/graph/graph_traits_Polyhedron_3.h>
+#include <CGAL/boost/graph/properties_Polyhedron_3.h>
 #include <CGAL/Nef_3/SNC_point_locator.h>
 #include <CGAL/assertions.h>
 

